packages/mask/src/plugins/Pets/SNSAdaptor/PetDialog.tsx

```diff
@@ -0,0 +1,283 @@
+
+    const [collection, setCollection] = useState<FilterContract>(initCollection)
+    const [isCollectionsError, setCollectionsError] = useState(false)
+
+    const [metaData, setMetaData] = useState<PetMetaDB>(initMeta)
+    const [isImageError, setImageError] = useState(false)
+    const [isTipShow, setTipShow] = useState(false)
+    const [holderChange, setHolderChange] = useState(true)
+
+    useEffect(() => {
+        if (!open) {
+            setMetaData(initMeta)
+            setCollection(initCollection)
+        }
+    }, [open])
+
+    useEffect(() => {
+        Promise.all(nfts.map((i) => PluginPetRPC.getAssetContract(i.contract, chainId))).then((lists) => {
+            setExtraData(lists)
+        })
+    }, [nfts])
+
+    const saveHandle = async () => {
+        if (!collection.name) {
+            setCollectionsError(true)
+            return
+        }
+        if (!metaData.image) {
+            setImageError(true)
+            return
+        }
+        const chosedToken = collection.tokens.find((item) => item.mediaUrl === metaData.image)
+        const meta = { ...metaData }
+        meta.userId = user.userId
+        meta.contract = collection.contract
+        meta.tokenId = chosedToken?.tokenId ?? ''
+        await PluginPetRPC.saveEssay(user?.address, meta, user?.userId ?? '')
+        setTipShow(true)
+        closeDialog()
+        setTimeout(() => {
+            setTipShow(false)
+        }, 2000)
+    }
+
+    const onCollectionChange = (v: string) => {
+        nfts.forEach((y) => {
+            if (y.name === v) {
+                setCollection(y)
+            }
+        })
+        setCollectionsError(false)
+    }
+
+    const onImageChange = (v: string) => {
+        setMetaData({ ...metaData, image: v })
+        setImageError(false)
+    }
+
+    const setMsgValueCheck = (v: string) => {
+        if (v.length <= 100) {
+            setMetaData({ ...metaData, word: v })
+        }
+    }
+
+    const imageChose = useMemo(() => {
+        if (!metaData.image) return ''
+        const imageChosed = collection.tokens.find((item) => item.mediaUrl === metaData.image)
+        return imageChosed?.mediaUrl
+    }, [metaData.image])
+
+    const renderImg = (contract: string) => {
+        const imgItem = extraData.filter((i) => i.address.toLowerCase() === contract.toLowerCase())
+        return <img className={classes.thumbnail} src={imgItem[0]?.image_url ?? ''} />
+    }
+
+    const paperComponent = (children: ReactNode | undefined) => (
+        <Box
+            style={{
+                boxShadow: '0 0 5px #ccc',
+                backgroundColor: '#FFFFFF',
+                marginBottom: 10,
+            }}>
+            {children}
+        </Box>
+    )
+
+    return (
+        <>
+            <InjectedDialog
+                open={open}
+                onClose={closeDialog}
+                title={t('plugin_pets_dialog_title')}
+                style={{ boxShadow: '0 0 5px #ccc' }}>
```

Comment:
Avoid a dead code color. The UI should be able to react to the theme configuration.